sympy/geometry/ellipse.py

```diff
@@ -34,6 +34,8 @@
 
 import random
 
+x = Dummy('x', real=True)
+y = Dummy('y', real=True)
```

Comment:
Would it be better to use more anonymous dummies `Dummy()` for anywhere where users seeing dummies should be a bug?
I think that it confuses, if such bug exist, at least in jupyter.

Reply:
What about the last commit where a generic name is used. I have found debugging to be a little difficult at times when trying to track down a generic "dummy_123" or "x". Now if the symbol appears it will be obvious where it was generated.

Reply:
If a Dummy is returned to a user, it should use a mathematically meaningful name (typically a single letter name like x or t). I agree it's useful to name internal dummies for debugging.

Reply:
None of these should be returned.

Reply:
I guess I misunderstood. I thought these were dummies that are returned to the user as a new parameter in the solution. If they are only used internally, then I agree that they can be reused. For dummies like that, having a long (but also unique) name like `Dummy('geometry_internal')` would be best, so that it's very obvious where it came from if it ever does leak out.